.travis.yml

```diff
@@ -29,6 +29,8 @@ notifications:
       - mowla@adobe.com
       - khoury@adobe.com
       - aahardy@adobe.com
+      - alloy-engs@adobe.com
```

Comment:
You can now drop some of the entries that are covered by these new groups right?

Reply:
Thank you @liljenback. Yes; I just wanted to test sending emails to a DL and make sure it works :)